flink-test-utils-parent/flink-test-utils-junit/src/test/java/org/apache/flink/testutils/junit/RetryOnExceptionExtensionTest.java

```diff
@@ -37,53 +41,77 @@
 @ExtendWith(RetryExtension.class)
 class RetryOnExceptionExtensionTest {
 
-    private static final int NUMBER_OF_RUNS = 3;
+    private static final int NUMBER_OF_RETRIES = 3;
 
-    private static int runsForSuccessfulTest = 0;
+    private static final HashMap<String, Integer> methodRunCount = new HashMap<>();
 
-    private static int runsForTestWithMatchingException = 0;
+    private static final HashMap<String, Runnable> verificationCallbackRegistry = new HashMap<>();
```

Comment:
nit: Generally, it's a good practice to use interfaces instead of classes for types. But that doesn't make much of a difference for private fields. :shrug: